java/cdk/src/main/java/sleeper/cdk/stack/bulkimport/PersistentEmrBulkImportStack.java

```diff
@@ -217,6 +217,15 @@ private static void configureJobStarterFunction(IFunction bulkImportJobStarter)
                 .effect(Effect.ALLOW)
                 .resources(Lists.newArrayList("*"))
                 .build());
+
+        bulkImportJobStarter.addToRolePolicy(PolicyStatement.Builder.create()
+                .sid("CreateCleanupRole")
+                .actions(Lists.newArrayList("iam:CreateServiceLinkedRole", "iam:PutRolePolicy"))
+                .resources(Lists.newArrayList("arn:aws:iam::*:role/aws-service-role/elasticmapreduce.amazonaws.com*/AWSServiceRoleForEMRCleanup*"))
+                .conditions(Map.of("StringLike", Map.of("iam:AWSServiceName",
+                        Lists.newArrayList("elasticmapreduce.amazonaws.com",
+                                "elasticmapreduce.amazonaws.com.cn"))))
+                .build());
```

Comment:
From the EMR docs it looks like this policy is needed at the point when you create the cluster:

https://docs.aws.amazon.com/emr/latest/ManagementGuide/using-service-linked-roles.html

For persistent EMR that's done by the CDK, so it sounds like it's only needed on the job starter function for non-persistent EMR, ie. EmrBulkImportStack?

Reply:
I have moved this to EmrBulkImportStack